common/src/main/java/com/wynntils/utils/render/FontRenderer.java

```diff
@@ -326,10 +326,7 @@ public void renderTexts(PoseStack poseStack, float x, float y, List<TextRenderTa
         float currentY = y;
         for (TextRenderTask line : lines) {
             renderText(poseStack, x, currentY, line);
-            // If we ask Mojang code the line height of an empty line we get 0 back so replace with space
```

Comment:
Are you sure line is never empty? Or is the bug fixed in Vanilla?

Reply:
Edit, ah, I see, the check has moved.

Reply:
It's been integrated below.